locales/en/collections.json

```diff
@@ -49,6 +49,14 @@
   "download_csv_incoming_moves": "Download moves (.csv)",
   "download_csv_single_requests": "Download moves (.csv)",
   "labels": {
+    "vehicle_registration": "Vehicle registration",
+    "awaiting_vehicle": "Awaiting vehicle",
+    "expected_time_incoming": "Expected arrival time",
+    "expected_time_outgoing": "Expected collection time",
+    "no_expected_time": "Not provided",
+    "complete_incoming": "Arrived",
+    "complete_outgoing": "Left custody",
+    "people": "people",
```

Comment:
Should this be lowercase? Looking at the screenshots, I think it's upper, but very likely missed something.

Reply:
I think this might be an old one that is no longer used...